### PR TITLE
PR for 2600 config changes 

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
PR for 2600 config changes

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: libraries/resource_providers/api2600/c7000/enclosure_provider.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline libraries/resource_providers/api2600/c7000/enclosure_provider.rb]: change copyright year for all files

File: libraries/resource_providers/api2600/synergy/hypervisor_manager_provider.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2600
+    module Synergy
+      # HypervisorManager API2600 Synergy provider
+      class HypervisorManagerProvider < OneviewCookbook::API2400::Synergy::HypervisorManagerProvider
```
Comment (VenkateshRavula) [inline libraries/resource_providers/api2600/synergy/hypervisor_manager_provider.rb:16]: what about force param.? Is that working in chef without changing library file..?

File: libraries/resource_providers/image_streamer/api2010/deployment_plan_provider.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2010 Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline libraries/resource_providers/image_streamer/api2010/deployment_plan_provider.rb]: this copyright year is our script issue. Please take care of these manually

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -119,6 +119,10 @@
     OneviewSDK::Client.new(url: 'https://oneview.example.com', user: 'Administrator', password: 'secret123', api_version: 2400)
   end
 
+  let(:client2600) do
```
Comment (VenkateshRavula) [inline spec/spec_helper.rb:122]: add an entry for i3s also

File: libraries/resource_providers/api2600.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline libraries/resource_providers/api2600.rb]: update CopyRights

File: libraries/resource_providers/api2600/c7000.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline libraries/resource_providers/api2600/c7000.rb]: same

File: libraries/resource_providers/api2600/c7000/connection_template_provider.rb [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline libraries/resource_providers/api2600/c7000/connection_template_provider.rb]: same

**Review by AsisBagga — APPROVED** [review]
LGTM

File: spec/spec_helper.rb (hunk 4 above)
Comment (yuvirani) [reply spec/spec_helper.rb:122]: Done

File: libraries/resource_providers/api2600/synergy/hypervisor_manager_provider.rb (hunk 2 above)
Comment (yuvirani) [reply libraries/resource_providers/api2600/synergy/hypervisor_manager_provider.rb:16]: Yes, just added the param in examples

File: libraries/resource_providers/image_streamer/api2010.rb [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# (c) Copyright 2010 Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline libraries/resource_providers/image_streamer/api2010.rb]: change copyright year

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**